lib/Targeting/ActionHandler/AssignTargetGroup.php

```diff
@@ -19,29 +19,21 @@
 
 use Pimcore\Model\Tool\Targeting\Rule;
 use Pimcore\Model\Tool\Targeting\TargetGroup;
-use Pimcore\Targeting\ConditionMatcherInterface;
 use Pimcore\Targeting\Model\VisitorInfo;
 use Pimcore\Targeting\Storage\TargetingStorageInterface;
 
 class AssignTargetGroup implements ActionHandlerInterface
 {
     const STORAGE_KEY = 'tg';
 
-    /**
-     * @var ConditionMatcherInterface
-     */
-    private $conditionMatcher;
-
     /**
      * @var TargetingStorageInterface
      */
     private $storage;
 
     public function __construct(
-        ConditionMatcherInterface $conditionMatcher,
```

Comment:
@blankse CMF extends this class and throws exception  after this change:
![image](https://user-images.githubusercontent.com/5137917/146738777-5a8d73ae-10e4-4350-bf25-8819e72f7306.png)
could you take care of that on CMF bundle? thanks!

Reply:
@dvesh3 I added it again with comment. Created an PR for CMF (https://github.com/pimcore/customer-data-framework/pull/247) so it can removed in Pimcore 11.